### DISCUSSION TITLE
Gopro9WifiCommands

### DISCUSSION THREAD

**Discussion by yeralexey** [KonradIT/goprowifihack#290] [Q&A]
Hi  @KonradIT !

First of all - great thanks for your great job. Automating my gopro happily now.

Have few questions about creating commits to this repository and pull requests to it.

1. I have Hero9, with downshifted 1.22 firmware for it. I installed your gopro-py-api, and see some requests not mentioned here, but the ones, that do work with HERO9, they are mentioned in other cameras folders, thow. Should i contribute them to this renository to Hero9 folder file or not? () If they are tested and do work with Hero9?

2. Some requests are mentioned in official documentation, and even in description, but not in file body(for example - `/gp/gpControl/status` , `http://10.5.5.9/gp/gpControl/command/mode?p=1`), what will be your opinion if i add it to file?

So, the whole idea, actually, is about to create an ability to download a whole bunch of tested camera commands with something like this: https://nekobin.com/xelobahubi.py , so to create an ability to perform different scenarios, like timelapses. This approach may be even Lua scripts, or Bash scripts, that coulb be run on routers with OpenWrt with GoPro wified to it... It woul be goot so whole bunch to be in one place, so your repository seems bo be best for it))

Thanks.

**Comment by KonradIT** [comment]
1. Yeah go ahead and make a PR, I don't have a HERO9 to test, so please document everything about how they work (response, any hangs, etc...)
2. Sure
